run_aprime.bash

```diff
@@ -353,35 +343,48 @@ if [ ! -d $www_dir/$plots_dir_name ]; then
 else
   rm -f $www_dir/$plots_dir_name/*
 fi
+# Check on MPAS-Analysis www_dir, permissions included
+# Create it if it does not exist, purge it if it does
+export mpas_www_link=mpas-analysis # mpas_www_link is used by generate_html to define
+                                   # the html link to the MPAS-Analysis web interface
+if [ ! -d $www_dir/$mpas_www_link ]; then
+  mkdir -p $www_dir/$mpas_www_link
+  chmod -R ga+rX $www_dir/$mpas_www_link
```

Comment:
I don't think this recursive chmod is useful here because you just created the directory.  Once it is populated, you will need to do a recursive chmod on the contents.  But it doesn't do any harm.